srcpkgs/mozjs68/template

```diff
@@ -0,0 +1,71 @@
+# Template file for 'mozjs68'
+pkgname=mozjs68
+version=68.8.0
+revision=1
+wrksrc="firefox-${version}"
+build_wrksrc=js/src
+build_style=gnu-configure
+hostmakedepends="perl python pkg-config automake autoconf213 autoconf-archive
+ which python3"
+makedepends="icu-devel libffi-devel nspr-devel python-devel readline-devel zlib-devel
+ python3-devel"
+depends="nspr>=4.19"
+short_desc="Mozilla JavaScript interpreter and library (68.x series)"
+maintainer="Enno Boland <gottox@voidlinux.org>"
+license="MPL-2.0"
+homepage="https://www.mozilla.org/js/"
+distfiles="${MOZILLA_SITE}/firefox/releases/${version}esr/source/firefox-${version}esr.source.tar.xz"
+checksum=fa5b2266d225878d4b35694678f79fd7e7a6d3c62759a40326129bd90f63e842
+patch_args="-Np1"
+CXXFLAGS="-Wno-class-memaccess"
+LDFLAGS+=" -Wl,-z,stack-size=1048576"
+
+if [ "$XBPS_TARGET_NO_ATOMIC8" ]; then
+	makedepends+=" libatomic-devel"
+	LDFLAGS+=" -latomic"
+fi
+
+do_configure() {
+	local _args
+
+	if [ "$CROSS_BUILD" ]; then
+		export HOST_CC=cc
+		export TARGET_CC="${CC}"
+		export HOST_CFLAGS="-Os"
+		export HOST_CXXFLAGS="-Os"
+		_args+=" --target=$XBPS_CROSS_TRIPLET --enable-linker=bfd"
+	fi
+
+	autoconf-2.13 old-configure.in > old-configure
+
+	touch ${wrksrc}/js/src/configure
+	touch ${wrksrc}/js/src/old-configure
+	SHELL=/bin/bash PYTHON=/usr/bin/python2 ./configure --prefix=/usr \
+		--disable-jemalloc --disable-optimize --enable-ctypes \
+		--enable-readline --enable-shared-js --enable-system-ffi \
+		--enable-tests --with-intl-api --with-system-icu --with-system-nspr \
+		--enable-unaligned-private-values  --enable-hardening \
+		--enable-release --with-system-zlib --target=${XBPS_TRIPLET} \
+		--host=${XBPS_TRIPLET} \
+		${_args}
+}
+
+do_check() {
+	dist/bin/jsapi-tests --format=none --exclude-random basic
+}
+
+post_install() {
+	# Remove unneeded static library
+	rm -f "${DESTDIR}"/usr/lib/*.ajs
```

Comment:
I'll move it to `-devel` instead.

Reply:
I removed it due to this https://gitlab.gnome.org/GNOME/jhbuild/issues/55
mozjs60 would probably still be needed for `cjs` ( it depends on mozjs52 atm)

Reply:
Turns out even `mozjs52-devel` installs the library with same name.